giraph-core/src/main/java/org/apache/giraph/graph/RetryableJobProgressTrackerClient.java

```diff
@@ -86,6 +93,14 @@ public RetryableJobProgressTrackerClient(GiraphConfiguration conf) throws
     resetConnection();
   }
```

Comment:
Do we need this now?

Reply:
@aanchal204, are you referring to the constructor? I left it there as we may still want to use it as we used to.

Reply:
Alright, makes sense!